gnomad/utils/liftover.py

```diff
@@ -13,193 +13,148 @@
-    else:
-        out_name = basename(path).split(".")[0]
-        out = f"{dirname(path)}/{out_name}_lift"
-        return f"{out}.ht" if is_table else f"{out}.mt"
+GRCH38_TO_GRCH37_CHAIN = "gs://hail-common/references/grch38_to_grch37.over.chain.gz"
+"""
+Path to chain file required to lift data from GRCh38 to GRCh37.
+"""
 
 
-def get_liftover_genome(t: Union[hl.MatrixTable, hl.Table]) -> list:
+def get_liftover_genome(
+    t: Union[hl.MatrixTable, hl.Table]
+) -> Tuple[hl.genetics.ReferenceGenome, hl.genetics.ReferenceGenome]:
     """
-    Infers genome build of input data and assumes destination build. Prepares to liftover to destination genome build
+    Infers reference genome build of input data and assumes destination reference genome build. 
+
+    Adds liftover chain to source reference genome and sequence to destination reference genome.
+    Returns tuple containing both reference genomes in preparation for liftover.
 
-    :param t: Input Table or MatrixTable
-    :return: List of source build (with liftover chain added) and destination build (with sequence loaded)
+    :param t: Input Table or MatrixTable.
+    :return: Tuple of source reference genome (with liftover chain added) 
+        and destination reference genome (with sequence loaded)
     """
 
-    logger.info("Inferring build of input")
-    build = get_reference_genome(t.locus).name
+    logger.info("Inferring reference genome of input...")
+    input_build = get_reference_genome(t.locus).name
+    source = hl.get_reference(input_build)
 
-    logger.info(
-        "Loading reference genomes, adding chain file, and loading fasta sequence for destination build"
-    )
-    if build == "GRCh38":
-        source = hl.get_reference("GRCh38")
+    logger.info("Loading fasta sequence for destination build...")
+    if input_build == "GRCh38":
         target = hl.get_reference("GRCh37")
-        chain = "gs://hail-common/references/grch38_to_grch37.over.chain.gz"
-        target.add_sequence(
-            "gs://hail-common/references/human_g1k_v37.fasta.gz",
-            "gs://hail-common/references/human_g1k_v37.fasta.fai",
-        )
+        chain = GRCH38_TO_GRCH37_CHAIN
+
     else:
-        source = hl.get_reference("GRCh37")
         target = hl.get_reference("GRCh38")
-        chain = "gs://hail-common/references/grch37_to_grch38.over.chain.gz"
-        target.add_sequence(
-            "gs://hail-common/references/Homo_sapiens_assembly38.fasta.gz",
-            "gs://hail-common/references/Homo_sapiens_assembly38.fasta.fai",
+        chain = GRCH37_to_GRCH38_CHAIN
+
+    logger.info("Adding liftover chain to input build...")
+    if source.has_liftover():
+        logger.warning(
+            f"Source reference build {source.name} already has a chain file!\
+            Using whichever chain has already been added."
+        )
+    else:
+        source.add_liftover(chain, target)
+
+    return (source, add_reference_sequence(target))
```

Comment:
I left this as a tuple because I couldn't remember if I needed to return the source ... I can remove the parentheses and change this to just return the target genome. what do you think?